website/static/scripts/tools.js

```diff
@@ -2,15 +2,17 @@
   var $currentItem;
   var $currentNav;
   var $steps = $(".step");
+  var $stepHidden = $(".step-hidden");
 
   function reset() {
     if ($currentItem) $currentItem.hide();
     if ($currentNav) $currentNav.removeClass("active");
     $currentItem = $currentNav = null;
     $steps.removeAttr("style");
+    $stepHidden.removeAttr("style");
   }
 
-  $(".btn-group .btn").click(function () {
+  $(".tools-group .tools-button").click(function () {
```

Comment:
It should be easy to convert these to non-jquery so we don't have to load it right?